tests/greenheart/test_openmdao.py

```diff
@@ -18,10 +18,10 @@
 wind_resource_file = Path(__file__).absolute().parent.parent.parent / "resource_files" / "wind" / "35.2018863_-101.945027_windtoolkit_2012_60min_80m_100m.srw"
 floris_input_filename = Path(__file__).absolute().parent / "inputs" / "floris_input.yaml"
 hopp_config_filename = Path(__file__).absolute().parent / "inputs" / "hopp_config.yaml"
-hopp_config_steel_ammonia_filename = Path(__file__).absolute().parent / "test_hydrogen" / "input_files" / "plant" / "hopp_config.yaml"
-greenheart_config_onshore_filename = Path(__file__).absolute().parent / "test_hydrogen" / "input_files" / "plant" / "greenheart_config_onshore.yaml"
-turbine_config_filename = Path(__file__).absolute().parent / "test_hydrogen" / "input_files" / "turbines" / "osw_18MW.yaml"
-floris_input_filename_steel_ammonia = Path(__file__).absolute().parent / "test_hydrogen" / "input_files" / "floris" / "floris_input_osw_18MW.yaml"
+hopp_config_steel_ammonia_filename = Path(__file__).absolute().parent / "input_files" / "plant" / "hopp_config.yaml"
+greenheart_config_onshore_filename = Path(__file__).absolute().parent / "input_files" / "plant" / "greenheart_config_onshore.yaml"
+turbine_config_filename = Path(__file__).absolute().parent / "input_files" / "turbines" / "osw_18MW.yaml"
+floris_input_filename_steel_ammonia = Path(__file__).absolute().parent / "input_files" / "floris" / "floris_input_osw_18MW.yaml"
 rtol = 1E-5
 
 class TestBoundaryDistanceComponent(unittest.TestCase):
```

Comment:
Rewrite as pytests?

Reply:
Right, good catch. I told you I would get this into this PR and forgot. I'll try to get to it soon.

Reply:
I have adjusted the test style in `test_openmdao.py` to follow the function pytest approach